internal/provider/harness_k3s_resource.go

```diff
@@ -328,14 +330,28 @@ func (r *HarnessK3sResource) Create(ctx context.Context, req resource.CreateRequ
 
 	kopts = append(kopts, k3s.WithNetworks(networks...))
 
-	harness, err := k3s.New(data.Id.ValueString(), r.store.cli, kopts...)
+	id := data.Id.ValueString()
+	configVolumeName := id + "-config"
+
+	_, err := r.store.cli.VolumeCreate(ctx, volume.CreateOptions{
+		Labels: map[string]string{"owner": "imagetest"},
```

Comment:
humble request to hoist this into a constant somewhere, since its also defaulted in `NewDocker()` to something different.

Maybe something similar to what we did for melange: https://github.com/chainguard-dev/melange/blob/165db39be30913d0a1c98258ba4e1451d3002d45/pkg/container/docker/docker_runner.go#L129-L132

Reply:
Updated!